surveys/models.py

```diff
@@ -13,6 +13,17 @@ class FacilitatorSurveyResponse(models.Model):
     response = models.TextField() #This will be a JSON value
     responded_at = models.DateTimeField()
 
+    def get_response_field(self, question_id):
+        response = json.loads(self.response)
+        answers = response['answers']
+        return next((answer for answer in answers if answer["field"]["id"] == question_id), None)
```

Comment:
I've never used JMESPath before but it looks promising, if we're going to be doing much more with the Typeform survey results I think it'd be worth using a tool for it because it's a pain in the butt to extract the data.